_gsdocs-proposals/2020/proposal_AllpixSquaredRestructureDocs.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Generic Proposal Title
+layout: gsoc_proposal
```

Comment:
The layout must read: gsdocs_proposal

Reply:
Ups, this I missed. It's also wrong in your example proposal, I fixed it there, too.